src/Repository/CommentRepository.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\Comment;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Persistence\ManagerRegistry;
+
+/**
+ * @method Comment|null find($id, $lockMode = null, $lockVersion = null)
+ * @method Comment|null findOneBy(array $criteria, array $orderBy = null)
+ * @method Comment[]    findAll()
+ * @method Comment[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+class CommentRepository extends ServiceEntityRepository
+{
+    public function __construct(ManagerRegistry $registry)
+    {
+        parent::__construct($registry, Comment::class);
+    }
+
+    /**
+     * @return Comment[] Returns an array of Comment objects
+     */
+    public function commentForum(): array
+    {
+        $query = $this->getEntityManager()->createQuery("SELECT c FROM App\Entity\Comment c WHERE c.figure = 'NULL' ");
```

Comment:
Cela devrait plus ressemblais à :
$query = $this->createQueryBuilder('c')
    ->where('c.figure  IS NULL')
    ->getQuery();

la forme et plus consise. 
De plus en SQL NULL est un type batarre et on n'est jamais certain de la réponse si on utilise =